src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -471,4 +474,9 @@ public SourcePath getObjCGeneratedHeaderPath() {
   public SourcePath getOutputPath() {
```

Comment:
Should we remove "Swift module metadata" from the comment above?

Reply:
No, the comment is still legit.